src/common/components/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import { FunctionComponent } from 'react';
+
+interface Props {
+  label: string;
+  onClick?: () => void;
```

Comment:
Pls create src/common/types/functions.ts file and put there interfaces of function types, e.g. 
`() => void 
`could be a VoidFunction,

`(param:T)=> void
`is a consumer function
(A Consumer is a functional interface that accepts a single input and returns no output.)